IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -212,10 +304,15 @@ @interface MSIDMacKeychainTokenCache ()
 @property MSIDMacCredentialStorageItem *appStorageItem;
 @property MSIDMacCredentialStorageItem *sharedStorageItem;
 @property MSIDCacheItemJsonSerializer *serializer;
+@property (readwrite, nonnull) id accessForSharedBlob;
+@property (readwrite, nonnull) id accessForNonSharedBlob;
+
 @end
 
 @implementation MSIDMacKeychainTokenCache
 
+#define CFReleaseNull(CF) { CFTypeRef _cf = (CF); if (_cf) CFRelease(_cf); CF = NULL; }
```

Comment:
this has been already defined somewhere else. Can we reuse?

Reply:
Addressed. Moved this macro to MSIDKeychainUtil.h which is shared by all keychain code for iOS and macOS.